<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaggicSegmentationDialog</class>
 <widget class="QDialog" name="MaggicSegmentationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaggicVision</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/icon/icon.png</normaloff>:/image/icon/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>91</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="entitiesLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Entities</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_2">
      <property name="text">
       <string>2</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_3">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_4">
      <property name="text">
       <string>4</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_5">
      <property name="text">
       <string>5</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_6">
      <property name="text">
       <string>6</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="entitiesButton_7">
      <property name="text">
       <string>7</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>511</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Extreme Saturation</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Multiplied Results</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Thresholded</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Segmented</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Details</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="visualizationLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>28</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="learningProgressBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>40</number>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Load default configuration (L)</string>
   </property>
   <property name="text">
    <string>Load Config</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="learnSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QSlider" name="minimumVerticalSlider">
   <property name="geometry">
    <rect>
     <x>661</x>
     <y>379</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="minimumLcdNumber">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Minimum threshold value</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSlider" name="maximumVerticalSlider">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>379</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="maximumLcdNumber">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Maximum threshold value</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reset (R)</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </widget>
  <widget class="QSlider" name="threshGrayHorizontalSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="threshGrayLcdNumber">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Grayscale Threshold value</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>255</number>
   </property>
  </widget>
  <widget class="QPushButton" name="playpauseButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Play/Pause (P)</string>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="applyLUTButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Apply the LUT (A)</string>
   </property>
   <property name="text">
    <string>Aplicar!</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vectorscopeCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Vectorscope (V)</string>
   </property>
   <property name="text">
    <string>Vectorscope</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="FilterCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Filter (F)</string>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NormalizationComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No Normalization</string>
    </property>
    <property name="icon">
     <iconset resource="iconTools/icontools.qrc">
      <normaloff>:/image/icon/35-turn-visibility-off-button.png</normaloff>:/image/icon/35-turn-visibility-off-button.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chromatic Normalization</string>
    </property>
    <property name="icon">
     <iconset resource="iconTools/icontools.qrc">
      <normaloff>:/image/icon/019-bee.png</normaloff>:/image/icon/019-bee.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vector Normalization</string>
    </property>
    <property name="icon">
     <iconset resource="iconTools/icontools.qrc">
      <normaloff>:/image/icon/017-strategy.png</normaloff>:/image/icon/017-strategy.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Weighted Normalization</string>
    </property>
    <property name="icon">
     <iconset resource="iconTools/icontools.qrc">
      <normaloff>:/image/icon/030-wand.png</normaloff>:/image/icon/030-wand.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>595</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="saturation">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>595</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="contrast">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>590</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
  <include location="iconTools/icontools.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MaggicSegmentationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MaggicSegmentationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
